RecoEgamma/EgammaTools/src/ElectronIDHelper.cc

```diff
@@ -0,0 +1,59 @@
+#include "RecoEgamma/EgammaTools/interface/ElectronIDHelper.h"
+
+#include <iostream>
+
+ElectronIDHelper::ElectronIDHelper(const edm::ParameterSet  & iConfig,edm::ConsumesCollector && iC):
+        eeRecHitInputTag_(iConfig.getParameter<edm::InputTag> ("EERecHits") ),
+        fhRecHitInputTag_(iConfig.getParameter<edm::InputTag> ("FHRecHits") ),
+        bhRecHitInputTag_(iConfig.getParameter<edm::InputTag> ("BHRecHits") ),
+        dEdXWeights_(iConfig.getParameter<std::vector<double> >("dEdXWeights")){
+
+    isoHelper_.setDeltaR(iConfig.getParameter<double>("electronIsoDeltaR"));
+    isoHelper_.setNRings(iConfig.getParameter<unsigned int>("electronIsoNRings"));
+    isoHelper_.setMinDeltaR(iConfig.getParameter<double>("electronIsoDeltaRmin"));
+
+    recHitsEE_ = iC.consumes<HGCRecHitCollection>(eeRecHitInputTag_);
+    recHitsFH_ = iC.consumes<HGCRecHitCollection>(fhRecHitInputTag_);
+    recHitsBH_ = iC.consumes<HGCRecHitCollection>(bhRecHitInputTag_);
```

Comment:
the helper apparently works only for HGCAL.
Please add HGC or HGCal to the name for more intuitive naming.